zipkin-cassandra/src/main/scala/com/twitter/cassie/Cluster.scala

```diff
@@ -59,12 +59,12 @@ class Cluster(seedHosts: Set[String], seedPort: Int, stats: StatsReceiver, trace
    * Returns a  [[com.twitter.cassie.KeyspaceBuilder]] instance.
    * @param name the keyspace's name
    */
-  def keyspace(name: String): KeyspaceBuilder = {
+  def keyspace(name: String, username: String, password: String): KeyspaceBuilder = {
```

Comment:
what happens to users of this library that don't require authentication? to support the old and new way, you could use method overloading and return a new type that `extends KeyspaceBuilder`, maybe `AuthenticatedKeyspaceBuilder`?